src/docs/Components/Cards/index.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Cards'
+order: 1
+description: 'A card is a navigation element that structures a group of related informations that must be understood on its own. It visually looks like a playing card.'
+status:
+  sketch: 'wip'
+  scss: 'wip'
+---
+
+> A card is a navigation element that structures a group of related informations that must be understood on its own. It visually looks like a playing card.
+
+A card is mostly a visualisation of the page that is to come behind the card itself.
+
+## Usages
+
+You will find two different cards versions. One is boxed, the other is not boxed.
+
+### Boxed version
+
+This version of the card is best for navigation and link elements between them throughout the user journey. For example, it's great to use to redirect the user to a specific category.
+
+### Unboxed version
+
+This version is more suited for informative content. It's best used at the end of a big page to redirect the user to an article for example.
+
```

Comment:
I'm not sure to get the differences between the boxed and unboxed version. 
Maybe it will make more sense with images.

Reply:
yeah I can try something using explicative images

Reply:
> 
> 
> I'm not sure to get the differences between the boxed and unboxed version.
> Maybe it will make more sense with images.

+1